scripts/measure_baseline.py

```
@@ -0,0 +1,198 @@
+    if args.rounds < 1:
+        parser.error("--rounds must be positive")
+
+    runs = run_benchmark_rounds(args.bin, args.filter + args.bench_args, args.cpus, args.rounds, args.raw_dir)
+    agg = aggregate_runs(runs)
+
+    if args.json:
+        metadata = {"command": [args.bin, "--bench", *args.filter, *args.bench_args],
+                    "cpus": args.cpus, "requested_rounds": args.rounds,
+                    "platform": platform.platform(), "cwd": os.getcwd(),
+                    "binary_sha256": hashlib.sha256(Path(args.bin).read_bytes()).hexdigest()}
+        revision = subprocess.run(["git", "rev-parse", "HEAD"], capture_output=True, text=True)
```

Ruff (0.16.3)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)